bin/command_rotator.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
```

Comment:
In the ts_mthexapod project you renamed the run_hexapod.py and command_hexapod.py scripts but you do not rename the command_rotator.py script. I suggest you do to have a homogeneous naming of your scripts. The same applies to the run_rotator.py script, which I see you did not rename either. Don't forget to update the files in the project that refer to these scripts.

Reply:
Good catch. I'll make these changes. Thanks!